app/helpers/verification_helper.rb

```diff
@@ -390,4 +400,19 @@ def has_active_resident_dependent?(person,dependent)
   def ridp_type_unverified?(ridp_type, person)
     ridp_type_status(ridp_type, person) != 'valid'
   end
+
+  private
+
+  def fetch_previous_states(verif_type)
+    verif_type&.type_history_elements&.pluck(:from_validation_status, :to_validation_status)&.flatten&.compact
+  end
+
+  def fetch_previous_states_history(verif_type, previous_states)
```

Comment:
Minor nitpick in the name here.  Could be slightly misleading as it is currently.  This doesn't really fetch states from the history tracks, but checks if any are equal to "outstanding".

Reply:
renamed to ```has_outstanding_history?```, let me know if you'd prefer anything else https://github.com/ideacrew/enroll/pull/4774/commits/6d80ef089df5eedc559b94e5d2342b53c0eb98c5#diff-aad1e1905cc0f848564bc64f88daf76e2057c834703c121490f1cfc22e815ac4R413